pkg/server/endpoints/endpoints.go

```diff
@@ -187,7 +189,8 @@ func (e *Endpoints) ListenAndServe(ctx context.Context) error {
 	trustdomainv1.RegisterTrustDomainServer(tcpServer, e.APIServers.TrustDomainServer)
 	trustdomainv1.RegisterTrustDomainServer(udsServer, e.APIServers.TrustDomainServer)
 
-	// Register Health and Debug only on UDS server
+	// UDP only
```

Comment:
```suggestion
	// UDS only
```

Reply:
Good eyes, my IP acronyms are working against me. :)

I'm changing this in my next push, only because accepting your suggestion through this interface will force a merge into my next push, through two repositories, and just editing the file will avoid the extra git work.